packages/flutter/lib/src/widgets/table.dart

```diff
@@ -371,16 +372,23 @@ class _TableElement extends RenderObjectElement {
   @override
   void insertRenderObjectChild(RenderObject child, dynamic slot) {
     renderObject.setupParentData(child);
+    if (_lastRemovedParentDataInThisFrame != null) {
+      renderObject.setChild(_lastRemovedParentDataInThisFrame!.x!, _lastRemovedParentDataInThisFrame!.y!, child as RenderBox);
+      _lastRemovedParentDataInThisFrame = null;
+    }
   }
 
   @override
   void moveRenderObjectChild(RenderObject child, dynamic oldSlot, dynamic newSlot) {
   }
 
+  TableCellParentData? _lastRemovedParentDataInThisFrame;
   @override
   void removeRenderObjectChild(RenderObject child, dynamic slot) {
     final TableCellParentData childParentData = child.parentData! as TableCellParentData;
     renderObject.setChild(childParentData.x!, childParentData.y!, null);
+    _lastRemovedParentDataInThisFrame = childParentData;
+    SchedulerBinding.instance!.addPostFrameCallback((Duration duration) => _lastRemovedParentDataInThisFrame = null);
```

Comment:
this wouldn't work for cases where this is done on a PipelineOwner that isn't being run by the binding...

Reply:
Could you let me know in what scenarios the situation you mentioned will be used(or some documataions)?
 I am not familiar with this and want to learn it.